lte/gateway/docker/mme/Dockerfile.rhel8

```diff
@@ -354,8 +365,8 @@ FROM magma-mme-builder as magma-dev-mme
 
 # Copy the configuration file templates and mean to modify/generate certificates
 WORKDIR /magma-mme/bin
-RUN cp $C_BUILD/core/oai/oai_mme/mme oai_mme
-RUN cp $C_BUILD/sctpd/src/sctpd .
+RUN cp $MAGMA_ROOT/bazel-bin/lte/gateway/c/core/oai_mme .
```

Comment:
Could also use `mv` here

Reply:
@LKreutzer 

Here I would not do it. It will be used when doing the `target` image.